ar.com.pilas_engine.App.yaml

```diff
@@ -0,0 +1,59 @@
+app-id: ar.com.pilas_engine.App
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: pilas-engine
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=home
+modules:
+  - name: extras
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml'
+      - 'install -Dm644 ${FLATPAK_ID}.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg'
+      - 'install -Dm644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop'
+    sources:
+      - type: file
+        path: extras/ar.com.pilas_engine.App.metainfo.xml
+      - type: file
+        path: extras/ar.com.pilas_engine.App.svg
+      - type: file
+        path: extras/ar.com.pilas_engine.App.desktop
+      # FIXME upstream extra files and obtain them from the release archive:
+      # - type: archive
+      #   url: https://github.com/pilas-engine/pilas-engine/archive/refs/tags/v2.7.3.zip
+      #   sha256: e1acc76cbcc3e3a8a222f0efb0feeef5fd882afe93a046c202333f4c25f196fb
+  - name: wrapper
+    buildsystem: simple
+    build-commands:
+      - 'install pilas-wrapper.sh /app/bin/pilas-engine'
+    sources:
+      - type: script
+        dest-filename: pilas-wrapper.sh
+        commands:
+          - 'exec zypak-wrapper /app/pilas-engine/pilas-engine \"$@\"'
+  - name: pilas
+    buildsystem: simple
+    build-commands:
+      - 'mkdir --parents /app/pilas-engine'
+      - 'if [ -d pilas-engine-linux-x64 ]; then mv --target-directory=/app/pilas-engine pilas-engine-linux-x64/*; fi'
+      - 'if [ -d pilas-engine-linux-ia32 ]; then mv --target-directory=/app/pilas-engine pilas-engine-linux-ia32/*; fi'
+    sources:
+      - type: archive
+        url: https://github.com/pilas-engine/pilas-engine/releases/download/v2.7.3/pilas-engine-linux-64_bits.zip
+        sha256: e1ae6b4c12cb861e53bc5ecb3cb1411dcea7a5927cdfec2be24fc7c8783f95d6
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://github.com/pilas-engine/pilas-engine/releases/download/v2.7.3/pilas-engine-linux-32_bits.zip
+        sha256: 250a5da3eccff2b5087b5b04ce60af471ce4de3b76eeea66705f44926a18099e
+        only-arches:
+          - x86_32
```

Comment:
1. this is an invalid arch
2. intel 32 bits hasn't been support for a few years now.

(so you should remove it)